hub/start_hub.bal

```diff
@@ -31,6 +32,7 @@ public function main() returns error? {
     // Initialize the Hub
     _ = @strand {thread: "any"} start syncRegsisteredTopicsCache();
     _ = @strand {thread: "any"} start syncSubscribersCache();
+   
 
```

Comment:
Why is this new line and above regex import required ?

Reply:
done